accession/export_csv_utils.py

```diff
@@ -0,0 +1,71 @@
+from accession.models import Accession, Donor, Object
+
+
+def get_object_by_model(model):
```

Comment:
This is a very generic name for a very specific function. What kind of object am i getting back. If I wasn't privy to what this function was actually doing, I would probably assume this would return a model instance.

What if we named it something like `get_csv_config()` instead?